### DISCUSSION TITLE
建议在未来版本中添加<一键多方块结构>和<模组引导手册>

### DISCUSSION THREAD

**Discussion by WaterDouble** [IndustrialCraftMC/IndustrialCraft2#26] [【意见建议】Ideas]
在创造模式中研究核反应堆或一些大型工程时，总是需要费力地建造多方块结构，这就很需要<一键多方块结构>去解决这个世纪难题，我的建议是可以将其与<模组引导手册>捆绑在一起，作为<模组引导手册>中的一部分（<模组引导手册>中的<一键多方块结构>部分仅创造可用，生存模式可以使其“可视立体化”，有利于教新手玩家如何建造各种多方块结构并使其理解其中工作原理）。

再说说<模组引导手册>，这本手册可以参考其他生电模组，例如加入<章节><阶段><物品讲解>等阶段性内容教程，有利于引导新手或不怎么会玩的玩家，使其有一个明确的目标，更有动力玩下去，而不是双眼一抹黑（这里可以参考植物魔法）。其实许多玩家也并没有耐心去上网学习攻略（毕竟找攻略就是一件极为麻烦的事情，我真的深有体会，哭死），这就会导致会的是真会，但是又和新手玩家说不清楚原理和游玩路线，从而导致新手玩家不敢入坑，老玩家又只减不增的尴尬场面。而且有了手册或官方引导，才能算是一个真正成熟又成体系的生电模组（例如沉浸工程和机械动力）。

以上我所提到的<一键多方块结构>和<模组引导手册>部分实际上要想实现确实是一件很困难且乏味的事情，是两项很大的工程，我认为可以往后放一放，先把高版本迁移这项大工程完成并进行完善，高版本迁移项目接近完美后沉淀，再进行这两个大项目也不迟。

**Comment by AbbottMc** [comment]
首先感谢建议~
其次我想说的是，这些内容其实开发起来并不枯燥（至少对于我来说是这样的），我个人也很乐意看到这些内容逐步被添加到IC2中。
然后关于这两个问题，我分开说吧
- 引导手册：这的确是工业2一直以来的痛点之一，不过其实我个人会更希望将其和游戏内容结合得更紧密些，而不是丢一本书给玩家让玩家自己看，即便是“手册”这种解决方案本身，在我眼里看来也并非最优解，更别说当下越来越多模组已经使用了类似的方案作为模组指南。而关于“引导”的这块内容我在去年年末更新基岩版工业2的那段时间就已经有大致的设计了，目前还在完善整个系统的设计，或许在工业2现有的问题基本修复后会着手开始开发。这将会是个全新的系统，我觉得还是很值得大家小小的期待一下的（
- “类一键多方块结构功能”：见工业3（好吧本来不想多说的，但可以稍微透露一下，在目前的设计中这部分内容大概是工业3中的一个全新体系的一部分）
